lib/request.js

```diff
@@ -242,7 +242,7 @@ module.exports = {
    */
 
   get hostname() {
-    const host = this.host;
+    const { host } = this;
```

Comment:
Having `const { host = '' } = this;` would remove the need for the line below it